### ISSUE TITLE
Responsive arrows with Customer prevArrow/nextArrow

### ISSUE THREAD

**Issue by aorinevo** [kenwheeler/slick#2944]
> short description of the bug / issue, provide more detail below.

When binding existing dom elements to prevArrow/nextArrow properties, responsive property with arrows set to false has no effect.  Works when using default prevArrow/nextArrow.

====================================================================

###### `[ http://jsfiddle.net/aorinevo/fmo50w7n/1221/ ]`  

use this jsfiddle to reproduce your bug: 
http://jsfiddle.net/simeydotme/fmo50w7n/ 
we will likely close your issue without it.

====================================================================

#### Steps to reproduce the problem

1. set prevArrow and nextArrow to existing DOM elements
2. Set responsive array with two break points and set arrows false for one.
3. Resize window between break points and see that arrows are always displayed.

====================================================================

#### What is the expected behaviour?

Arrows should be slick-hidden for when breakpoint with arrows=false is hit.

====================================================================

#### What is observed behaviour?

Arrows are always shown.

====================================================================

#### More Details

- Which browsers/versions does it happen on? Chrome 58.0.3029.110
- Which jQuery/Slick version are you using? jQuery 1.11.3/Slick 1.6.0
- Did this work before? N/A

**Comment by jacobarriola** [comment]
I can also reproduce this issue. Thanks for creating this @aorinevo

**Comment by aorinevo** [comment]
@jacobarriola Have you had a chance to checkout the PR for the fix? https://github.com/kenwheeler/slick/pull/2945

**Comment by danbohea** [comment]
I get this behaviour with the default (non-custom) arrows.

**Comment by jacobarriola** [comment]
@aorinevo Yes, I saw the PR; it's not merged yet, right? I can't touch the src file b/c everything gets bundled at build for my project.

**Comment by aorinevo** [comment]
@jacobarriola Could provide some feedback on the proposed fix in the PR.  Does it work? Is there another approach?

I think this will help the authors/reviewers figure out what the next steps are for the PR.
